### PR TITLE
since the images for DownloadButtons are different heights, they can …

### LINKED ISSUE #51: Google play button squashed dimensions
Description:
The google play button is scaled weirdly and isn't the correct dimensions. I have attached screenshots of what it should look like and what it currently looks like on the site.

Reporter:
James Elgar 

Trello link:
https://trello.com/c/C5v7bfKp/139-google-play-button-squashed-dimensions

Comments:

### PR DESCRIPTION
**Pull request by tunangst**
resolves #51 
This is a quick and easy fix I made, I can find some way to scale the height based on the width if you like instead. I won't be able to start that until at least Wednesday though.

Sorry for any git complications, I am used to doing self projects instead of working on other people's projects :].

### CODE REVIEW

File: src/components/download_buttons/DownloadButtons.module.scss [hunk 1]
```diff
@@ -1,35 +1,87 @@
...
-    max-width: 100%;
-    width: 90%;
-  }
-}
-
-@media (max-width: $small) {
-  .downloadBtns {
+    margin: 2em 0;
+    height: 90px;
     & a {
```
Comment (JElgar) [inline src/components/download_buttons/DownloadButtons.module.scss:6]: I know this isn't part of your commit - but we don't actually need these ampersands
`& a` is the same as just having `a` (when nested and with a space) and I'd probably say not having them is a bit neater.
```suggestion
    a {
```
https://css-tricks.com/the-sass-ampersand/

File: src/components/download_buttons/DownloadButtons.module.scss [hunk 2]
```diff
@@ -1,35 +1,87 @@
...
+//   }
+// }
+
+// @media (max-width: $medium) {
+//   .downloadBtns {
+//     & a {
+//       max-height: 70px;
+//     }
+//   }
+// }
```
Comment (JElgar) [inline src/components/download_buttons/DownloadButtons.module.scss:87]: It's probably best to get rid of commented out code from the `dev` branch (if you want to keep a reference to it for future I work stick it on a separate branch)
```suggestion
```

File: src/components/download_buttons/DownloadButtons.module.scss [hunk 3]
```diff
@@ -1,35 +1,87 @@
...
-  .downloadBtns {
+    margin: 2em 0;
+    height: 90px;
     & a {
-      max-height: 50px;
+        &:nth-child(1) {
+            margin-right: 50px;
+        }
+    }
+    & img {
```
Comment (JElgar) [inline src/components/download_buttons/DownloadButtons.module.scss:11]:
```suggestion
    img {
```

File: src/components/download_buttons/DownloadButtons.module.scss [hunk 4]
```diff
@@ -1,35 +1,87 @@
...
+}
+@media (max-width: $small) {
+    .downloadBtns {
+        width: 100%;
+        height: 50px;
+    }
+}
+@media (max-width: 410px) {
+    .downloadBtns {
+        height: 40px;
```
Comment (JElgar) [inline src/components/download_buttons/DownloadButtons.module.scss:39]: I'm also a big fan of nesting all the things so I would say these media queries should go in the .downloadBtn block. Example:
```
.profile-pic {
  float: left;
  width: 250px;
  @media screen and (max-width: $break-small) {
    width: 100px;
    float: none;
  }
```
which compiles to:
```
.profile-pic {
  float: left;
  width: 250px;
}
@media screen and (max-width: 320px) {
  .profile-pic {
    width: 100px;
    float: none;
  }
}
```
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

**Comment by JElgar** [comment]
Sorry, @tunangst I just read your original comment! Don't worry about the git stuff you should just be able to accept the changes that I requested for those kinds of things from here. For the nesting, if you're happy with me doing it then I'll swap that out later today.

**Comment by stellenberger** [comment]
I am happy with this + your suggestions @JElgar, so if @tunangst gives your changes the thumbs up ill merge this. I also did a slight change in my campaigns branch regarding the buttons, but this looks like a better solution!

**Comment by tunangst** [comment]
Okay, I made the suggested changes in my forked repo. It looks like it added my commits here. Sorry to waste so much of your time commenting on my code, I will learn your coding preferences :]

**Comment by JElgar** [comment]
@tunangst No worries at all! It obviously just a preference thing and this whole code reviewing things a bit new so I'll be picking up on stuff that already exists in the repo! But also feel free to propose better ways of doing things or tell me I'm wrong with the changes I suggest.

**Pull request merged**